sdf/1.7/frame.sdf

```diff
@@ -1,16 +1,31 @@
 <!-- Frame -->
 <element name="frame" required="*">
-  <description>A frame of reference to which a pose is relative.</description>
+  <description>A frame of reference in which poses may be expressed.</description>
 
   <attribute name="name" type="string" default="" required="1">
-    <description>Name of the frame. This name must not match another frame defined inside the parent that this frame is attached to.</description>
+    <description>
+      Name of the frame. It must be unique whithin its scope (model/world),
+      i.e., it must not match the name of another frame or model within the same
+      scope.
+    </description>
   </attribute>
 
   <attribute name="attached_to" type="string" default="" required="*">
     <description>
-      Name of the link or frame to which this frame is attached.
-      If a frame is specified, recursively following the attached_to attributes
-      of the specified frames must lead to the name of a link, a model, or the world frame.
+      If specified, this frame is attached to the specified frame. The specified
+      frame must be within the same scope and may be defined implicitly, i.e.,
+      the name of any //frame, //model, or //link within the same scope may be
+      used.
+
+      If missing, this frame is attached to the containing scope's frame. Within
+      a //world scope this is the implicit world frame, and within a //model
+      scope this is the implicit frame defined by the model's canonical link.
+
+      A frame moves jointly with the frame it is @attached_to. This is different
+      from //pose/@relative_to. @attached_to defines how the frame is attached
+      to the scope's frame, while //pose/@relative_to defines how the frame's
+      pose is represented numerically. As a result, following the chain of
+      @attached_to attributes must always lead to the scope's frame.
```

Comment:
> following the chain of @attached_to attributes must always lead to the scope's frame.

based on the discussion above, I would rephrase as "following the chain of @attached_to attributes must always lead to a link, model, or world frame."

Reply:
Will change this depending on the outcome of above discussion.